### DISCUSSION TITLE
Github Packages 500 internal server error

### DISCUSSION THREAD

**Discussion by n-tdi** [community/community#47849] [Packages] [answered]
### Select Topic Area

Bug

### Body

I'm trying to push my package to maven with github packages and right now it's giving me a `internal error 500` status failure. Is this an issue on github thats going on right now? Was just working a few hours ago..

**Comment by lgnl248** [comment, marked as answer]
Yeah it's an issue with Github right now

**Reply by n-tdi** [reply]
Dang, such an inconvenience.

**Comment by Aaron-Zhao-Asurion** [comment]
https://www.githubstatus.com/

**Reply by n-tdi** [reply]
Interesting! That wasn’t there when I first looked.

**Reply by Aaron-Zhao-Asurion** [reply]
yeah, I checked as well. It wasn't reflected on the status page until much later. But I suspected it was a Github issue.

**Comment by oodafeisrael** [comment]
server error keeps popping up when trying to create a repo

**Comment by oodafeisrael** [comment]
server error 500  page keeps coming up when i tried creating a repository
